test/user.test.js

```diff
@@ -138,7 +138,7 @@ describe('User', function() {
       });
     });
 
-    it('Email is required', function(done) {
+    it('Email is required (email case-sensitivity on)', function(done) {
```

Comment:
While you are changing this line, please edit the test name to read as an English sentence.

```js
it('fails when the required email is missing (email case-sensitivity on)', function(done) {
  //...
});
```